lua/wikis/commons/MatchGroup/Input/Util.lua

```diff
@@ -437,6 +440,9 @@ function MatchGroupInputUtil.readPlayersOfTeam(teamName, manualPlayersInput, opt
 			faction = player.faction and Faction.read(player.faction) or nil,
 			index = playersIndex,
 		})
+		players[normalizedPageName].extradata = Table.merge(players[normalizedPageName].extradata or {}, {
+			customId = player.customId
+		})
```

Comment:
Would this be necessary/advisable?
```suggestion
		players[normalizedPageName].extradata = Logic.nilIfEmpty(Table.merge(players[normalizedPageName].extradata or {}, {
			customId = player.customId
		}))
```

Reply:
imho rather throw the nilIfEmpty on the 2nd table in the merge
if you have `players[normalizedPageName].extradata` be `{}` before the merge you make it nil and hence potentially break stuff on some wikis

unless the extradata is never set at all on any wiki (in which case you wouldn't need to merge to be gin with)

Reply:
I think this is what i'd actually want then, otherwise it'll still set the empty table which i wanted to avoid
```suggestion
		if player.customId then
			players[normalizedPageName].extradata = Table.merge(players[normalizedPageName].extradata or {}, {
				customId = player.customId
			})
		end
```